src/sections/Resume/Resume.jsx

```diff
@@ -0,0 +1,229 @@
+                  reference: "Nullam fringilla sapien ac, Helsinki, FI",
+                  text:
+                    "Lorem ipsum dolor sit amet consectetur adipiscing elit arcu imperdiet dignissim, felis ultrices nunc suspendisse ac risus hac turpis elementum."
+                },
+                {
+                  date: {
+                    from: "2018",
+                    to: "2019"
+                  },
+                  title: "Curae primis vehicula",
+                  reference: "Elementum quisque himenaeos, Paris, FR",
+                  text:
+                    "Lorem ipsum dolor sit amet consectetur adipiscing elit, nostra elementum ultrices condimentum turpis rutrum, malesuada rhoncus montes luctus pretium feugiat."
+                },
+                {
+                  date: {
+                    from: "2017",
+                    to: "2018"
+                  },
+                  title: "Vitae pulvinar aenean",
+                  reference: "Phasellus semper potenti, Zaragoza, ES",
+                  text:
+                    "Lorem ipsum dolor sit amet consectetur adipiscing elit, lectus sem eu sodales lacus sed, at velit integer netus aliquam libero."
+                }
+              ]}
+            />
+          </div>
+          <div className={classes.timeline}>
+            <div className={classes.subtitle}>
+              <SectionSubtitle subtitleText='EDUCATION' />
+            </div>
+            <TimelineMaker
+              timeline={[
+                {
+                  date: {
+                    from: "2016",
+                    to: "2017"
+                  },
+                  title: "Torquent ut",
+                  reference: "Pretium primis molestie, NY, USA",
+                  text:
+                    "Lorem ipsum dolor sit amet consectetur adipiscing elit, nunc quis ad vehicula tristique senectus, venenatis vulputate hendrerit sagittis diam duis."
+                },
+                {
+                  date: {
+                    from: "2014",
+                    to: "2016"
+                  },
+                  title: "Nostra nibh viverra",
+                  reference: "Diam nascetur, Berlin, DE",
+                  text:
+                    "Lorem ipsum dolor sit amet consectetur adipiscing elit curae taciti congue, eleifend faucibus duis accumsan in nulla metus posuere penatibus."
+                },
+                {
+                  date: {
+                    from: "2013",
+                    to: "2014"
+                  },
+                  title: "Mauris quam",
+                  reference: "Orci suspendisse, Warwick, UK",
+                  text:
+                    "Lorem ipsum dolor sit amet consectetur adipiscing elit, tristique posuere dictumst hac laoreet suspendisse, pretium curae mi morbi vel cras."
+                }
+              ]}
+            />
+          </div>
+        </div>
+        <div className={classes.skills}>
```

Comment:
I see the structures above and here to be somehow similar, a title and then a content. Can this be defined as a generic component and then pass the title and content as a prop? (maybe including some styles)